infrastructure-playbooks/vars/lv_vars.yaml

```diff
@@ -0,0 +1,24 @@
+# Path of device primed for LV creation.
+device: /dev/sdc
+
+# Since this playbook can be run multiple times across different devices, device_basename is to differenciate the names of journals or data lvs if the names are not changed
+device_basename: "{{ device.split('/')[-1] }}"
```

Comment:
could add an example to the comment, to show what it would look like, in this particular case.